src/backend/app/config.py

```diff
@@ -161,6 +161,11 @@ class Settings(BaseSettings):
 
     EXTRA_CORS_ORIGINS: Optional[str | list[str]] = None
 
+    @property
```

Comment:
This is pretty nice, pydantic computed isn't necessarily needed as we don't serialise the model, so a property works well here